### PR TITLE
2016Q4.feature.odrecinspect

### PR DESCRIPTION
**Pull request by chrberger**
* Adding inter-container time stamping check

### CODE REVIEW

File: odtools/odrecinspect/include/RecInspect.h [hunk 1]
```diff
@@ -69,6 +69,7 @@ namespace odrecinspect {
         private:
             std::map<int32_t, uint64_t> m_numberOfContainersPerType;
             std::map<int32_t, odcore::data::Container> m_latestContainersPerType;
+            std::map<int32_t, uint32_t> m_numberOfContainersInIncorrectTemporalOrderPerType;
```
Comment (HangYin2015) [inline odtools/odrecinspect/include/RecInspect.h:72]: Why do we have "InIn" here? Should it be "In"?

Comment (chrberger) [reply odtools/odrecinspect/include/RecInspect.h:72]: No, that member counts the number of containers that are **in in**correct temporal order.

Comment (HangYin2015) [reply odtools/odrecinspect/include/RecInspect.h:72]: OK. It looks like a typo for the first glance. But your explanation makes sense:)

**Review by HangYin2015 — APPROVED** [review]

**Comment by chrberger** [comment]
@HangYin2015 May I ask you for another review as I added a few changes?

**Review by HangYin2015 — APPROVED** [review]

**Pull request merged**